CHANGELOG.rst

```diff
@@ -2,6 +2,42 @@
 CHANGELOG
 =========
 
+2.9.0
+=====
+
+**ENHANCEMENTS**
+
+* Added support for heterogeneous instance types with Slurm Scheduler.
```

Comment:
Add support for multiple queues and multiple instance types feature with the Slurm scheduler.